src/bin/cargo/commands/uninstall.rs

```diff
@@ -37,3 +44,34 @@ pub fn exec(gctx: &mut GlobalContext, args: &ArgMatches) -> CliResult {
     ops::uninstall(root, specs, &values(args, "bin"), gctx)?;
     Ok(())
 }
+
+fn get_installed_crates() -> Vec<clap_complete::dynamic::CompletionCandidate> {
+    let gctx = match GlobalContext::default() {
+        Ok(gctx) => gctx,
+        Err(_) => return vec![],
+    };
+    let mut candidates = Vec::new();
+
+    // HACK: Is there a better way to get the root directory without ArgMatches?
```

Comment:
That is only relevant when the user explicitly overrode it on the command line.  The old completions didn't bother with it and we don't support context-aware completions like that, so its fine to drop this comment